modules/oxide/src/less/theme/content/footnotes/footnotes.less

```diff
@@ -0,0 +1,32 @@
+//
+// Footnotes
+//
+
+sup.mce-footnote a {
+  color: black;
+  text-decoration: none;
+}
+
+div.mce-footnotes {
+  hr {
+    margin-left: 0;
+    margin-right: auto;
+    width: 25%;
+  }
+
+  li > a:first-child {
```

Comment:
@ltrouton is there anything that stops users putting in an <a> tag into the footnotes section entry? If not, should we have a class on the `a` so that it is more restrictive about which `a` we are styling / removing?

Reply:
They can add a link but it can only be added to the editable span which is the second child of the `li`. The backlink anchor `a` tag is noneditable and should always be the first child of the `li`.

To be honest, adding a class to the backlink `a` would probably be helpful and the most robust approach

Reply:
Right. I misread `first-child` again. Very out of practice reading CSS selectors. I thought it was saying the first child of the `a` tag, not choose only the `a` tags that are the first children. But never rely on position like this, if you can control and annotate the DOM elements (as a rule)